docs/book/en/src/use_cases/kubernetes/kubernetes/kubernetes-containerd.md

```diff
@@ -1,119 +1,3 @@
-```
-
-Let's check the status to make sure that the cluster is running.
-
-```bash
-$ sudo cluster/kubectl.sh cluster-info
-
-# Expected output
-Cluster "local" set.
-User "myself" set.
-Context "local" created.
-Switched to context "local".
-Kubernetes control plane is running at https://localhost:6443
-CoreDNS is running at https://localhost:6443/api/v1/namespaces/kube-system/services/kube-dns:dns/proxy
-
-To further debug and diagnose cluster problems, use 'kubectl cluster-info dump'.
-```
-
-### A simple WebAssembly app
-
-[A separate article](../demo/wasi.md) explains how to compile, package, and publish a simple WebAssembly WASI program as a container image to Docker hub.
-Run the WebAssembly-based image from Docker Hub in the Kubernetes cluster as follows.
-
-```bash
-sudo cluster/kubectl.sh run -it --rm --restart=Never wasi-demo --image=wasmedge/example-wasi:latest --annotations="module.wasm.image/variant=compat-smart" --overrides='{"kind":"Pod", "apiVersion":"v1", "spec": {"hostNetwork": true}}' /wasi_example_main.wasm 50000000
-```
-
-The output from the containerized application is printed into the console.
-
-```bash
-Random number: 401583443
-Random bytes: [192, 226, 162, 92, 129, 17, 186, 164, 239, 84, 98, 255, 209, 79, 51, 227, 103, 83, 253, 31, 78, 239, 33, 218, 68, 208, 91, 56, 37, 200, 32, 12, 106, 101, 241, 78, 161, 16, 240, 158, 42, 24, 29, 121, 78, 19, 157, 185, 32, 162, 95, 214, 175, 46, 170, 100, 212, 33, 27, 190, 139, 121, 121, 222, 230, 125, 251, 21, 210, 246, 215, 127, 176, 224, 38, 184, 201, 74, 76, 133, 233, 129, 48, 239, 106, 164, 190, 29, 118, 71, 79, 203, 92, 71, 68, 96, 33, 240, 228, 62, 45, 196, 149, 21, 23, 143, 169, 163, 136, 206, 214, 244, 26, 194, 25, 101, 8, 236, 247, 5, 164, 117, 40, 220, 52, 217, 92, 179]
-Printed from wasi: This is from a main function
-This is from a main function
-The env vars are as follows.
-The args are as follows.
-/wasi_example_main.wasm
-50000000
-File content is This is in a file
-pod "wasi-demo-2" deleted
-```
-
-### A WebAssembly-based HTTP service
-
-[A separate article](../demo/server.md) explains how to compile, package, and publish a simple WebAssembly HTTP service application as a container image to Docker hub.
-Run the WebAssembly-based image from Docker Hub in the Kubernetes cluster as follows.
-
-```bash
-sudo cluster/kubectl.sh run --restart=Never http-server --image=wasmedge/example-wasi-http:latest --annotations="module.wasm.image/variant=compat-smart" --overrides='{"kind":"Pod", "apiVersion":"v1", "spec": {"hostNetwork": true}}'
-```
-
-Since we are using `hostNetwork` in the `kubectl run` command, the HTTP server image is running on the local network with IP address `127.0.0.1`.
-Now, you can use the `curl` command to access the HTTP service.
-
-```bash
-$ curl -d "name=WasmEdge" -X POST http://127.0.0.1:1234
-echo: name=WasmEdge
-```
-
-That's it!
+> This part has been moved to <https://wasmedge.org/docs/category/high-level-runtime-support>. Please use or new docs
```

Comment:
This part is here: https://wasmedge.org/docs/develop/deploy/kubernetes/kubernetes-containerd-crun